.vscode/extensions.json

```diff
@@ -1,6 +1,9 @@
 {
   "recommendations": [
     "ms-vscode.cpptools-extension-pack",
-    "ms-python.python"
+    "ms-python.python",
+    "gruntfuggly.todo-tree",
```

Comment:
I am uncertain it is advisable to add these optional extension recommendations as they may conflict with a user's existing extensions.

The `ms-vscode.cpptools-extension-pack` and `ms-python.python` extensions I included in the original commit are necessary for a clean install of Visual Studio Code to build and debug C++ and Python code.

Reply:
I had that feeling. I will remove those extra ones before merge. The intention was to provide an environment very beginner-friendly, to help new / inexperienced people get started. It is explained a bit better in the document I've just written and linked.